doc/release-notes/0.15/0.15.md

```diff
@@ -0,0 +1,150 @@
+
+:construction: These release notes are currently in draft
+
+
+## Binaries and supported environments
+
+OpenJ9 release 0.15 supports OpenJDK 8, 11, and 12. Binaries are available at the AdoptOpenJDK project:
+
+- [OpenJDK 8 with OpenJ9](https://adoptopenjdk.net/archive.html?variant=openjdk8&jvmVariant=openj9)
+- [OpenJDK 11 with OpenJ9](https://adoptopenjdk.net/archive.html?variant=openjdk11&jvmVariant=openj9)
+- [OpenJDK 12 with OpenJ9](https://adoptopenjdk.net/archive.html?variant=openjdk12&jvmVariant=openj9)
+
+All builds are tested against the OpenJ9 functional verification (FV) test suite, the OpenJDK test suites, and additional tests at AdoptOpenJDK.
+
+To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](https://eclipse.org/openj9/docs/openj9_support/index.html).
+
+
+## Notable changes in this release
+
+The following table covers notable changes in v0.14. Further information about these changes can be found in the [user documentation](https://www.eclipse.org/openj9/docs/version0.14/).
+
+<table cellpadding="4" cellspacing="0" summary="" width="100%" rules="all" frame="border" border="1"><thead align="left">
+<tr valign="bottom">
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Version / Platform</th>
+<th valign="bottom">Impact</th>
+</tr>
+</thead>
+<tbody>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/5598">#5598</a></td>
+<td valign="top">Performance improvements for JVMTI watched fields by default</td>
+<td valign="top">OpenJDK8 and later (Windows, macOS, and Linux (x86 only))</td>
+<td valign="top">Following successful results, the experimental -XX:[+|-]JITInlineWatches option is now enabled by default, providing
+performance improvements for JVMTI watched fields. </td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/5183">#5183</a></td>
+<td valign="top">Extended platform support for software-based pause-less garbage collection</td>
+<td valign="top">OpenJDK8 and later (IBM Z systems)</td>
+<td valign="top">Reduced garbage collection pause times when using -Xgc:concurrentScavenge with the gencon GC policy for Linux on Z systems. </td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/ibmruntimes/openj9-openjdk-jdk11/pull/164">#164</a></td>
+<td valign="top">ChaCha20 algorithm support </td>
+<td valign="top">OpenJDK11 only</td>
+<td valign="top">On Java 11, ChaCha20 and ChaCha20-Poly1305 algorithms can now use OpenSSL. </td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/54">#54</a></td>
+<td valign="top">OpenJ9 now supports all POSIX signals listed in [SunMiscSignalTest.java](https://github.ibm.com/runtimes/openj9-openjdk-jdk/blob/openj9/test/jdk/sun/misc/SunMiscSignalTest.java)</td>
```

Comment:
The link is to an ibm internal repo, use https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/openj9/test/jdk/sun/misc/SunMiscSignalTest.java

Reply:
Fixed